monkey/monkey_island/cc/server_setup.py

```diff
@@ -54,6 +58,7 @@ def run_monkey_island():
 
     connect_to_mongodb()
 
+    _configure_gevent_exception_handling(logger, Path(config_options.data_dir))
```

Comment:
Why pass `logger` as an argument here? Won't it just be able to use `logger` because of `logger = logging.getLogger(__name__)` above?